index.html

```diff
@@ -26,22 +28,151 @@
             don’t hesitate to contact me.</p>
             <nav class="icon-wrap">
                 <ul class="navicon">
-                   <li><a href="https://github.com/Zuwairanajma"><img class="github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
-                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class="linkedin" src ="PImages/Linkedin.png" alt = "Linkedin Icon"></a></li> 
-                   <li><a href="#"><img class ="angeli" src ="PImages/Vector3.png" alt="Angellist Icon"></a></li>
-                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class ="twitter" src ="PImages/Twitter.png" alt ="Twitter Icon"></a></li>
-                   <li><a href="#"><img class ="medium" src="PImages/Vector5.png" alt ="Medium Icon"></a></li>
+                   <li><a href="https://github.com/Zuwairanajma"><img class= "github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
+                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class= "linkedin"src="PImages/Linkedin.png" alt="Linkedin Icon"></a></li> 
+                   <li><a href="#"><img class= "angeli"src="PImages/VectorH.png" alt="Angellist Icon"></a> </li>
+                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class= "twitter" src="PImages/Twitter.png" alt="Twitter Icon"></a></li>
+                   <li><a href="#"><img class= "medium" src="PImages/Medium.png" alt="Medium Icon"></a></li>
+
                 </ul>
             </nav>
         </div>
     </section>
     <!-- Headline Section End -->
+  
+    <!-- Recent Works Section Start -->
+    <section id="My-Recent" class="My-Recent">
+        <div id="recent-works_container" class="recent-works">
+        <h1 id="my-work" class="my-work">My Recent Works</h1>
+      </div>
+
+        <div>
+        <img class = "placeholder" src="PImages/Img Placeholder.png" alt=""></div>
+            <div id="multip" class="multipost">
+                <h2 id="multi" class="multi">Multi-Post Stories</h2>
+                <p id="stories" class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <div id="buttons1" class="buttonss1">
+                <button type="button" class="btn btn-primary btn-lg">CSS</button>
+                <button type="button" class="btn btn-primary btn-lg">HTML</button>
+                <button type="button" class="btn btn-primary btn-lg">BootStrap</button>
+                <button type="button" class="btn btn-primary btn-lg">Ruby</button>
+            </div>
+            <buttons id="seeproject" class="btn2 btn-primary btn-lg">See Project</buttons>
+    </section>
+    <section class="cardworks" id="cards">
```

Comment:
- [ ] Kindly consider having all the cards in one section tag, the one you have already taken, so there is no need to have another section tag in order to have maintained and clean code.